great_expectations_provider/example_dags/example_great_expectations_dag.py

```diff
@@ -3,113 +3,208 @@
 """
 
 import os
+from pathlib import Path
+
 import airflow
 from airflow import DAG
-from great_expectations_provider.operators.great_expectations import (
-    GreatExpectationsOperator,
-    GreatExpectationsBigQueryOperator,
-)
+from great_expectations.core.batch import BatchRequest
+from great_expectations.data_context.types.base import (CheckpointConfig,
+                                                        DataContextConfig)
+
+from great_expectations_provider.operators.great_expectations import \
+    GreatExpectationsOperator
 
 default_args = {"owner": "Airflow", "start_date": airflow.utils.dates.days_ago(1)}
 
 dag = DAG(dag_id="example_great_expectations_dag", default_args=default_args)
```

Comment:
@josh-fell and I are working on some guidelines around DAG writing best practices, and two things we're trying to move away from are using `default_args` and this old way of defining a DAG object. The only arg you really need from `default_args` is `start_date`, which should be a static date (it doesn't really matter when, I usually just choose the date I am writing the DAG); using `days_ago` can cause problems.
Preferred ways of defining DAGs are using it as a [context manager or using the decorator](https://airflow.apache.org/docs/apache-airflow/2.0.0/concepts.html#context-manager).